community/hazelcast-enterprise/README.md

```diff
@@ -29,12 +23,79 @@ spec:
   - name: docker.io/hazelcast/*
 ```
 
+### PodSecurityPolicy Requirements
+
+This chart requires a PodSecurityPolicy to be bound to the target namespace prior to installation. Choose either a predefined PodSecurityPolicy or have your cluster administrator create a custom PodSecurityPolicy for you:
+
+* Predefined PodSecurityPolicy name: [`ibm-restricted-psp`](https://ibm.biz/cpkspec-psp)
+* Custom PodSecurityPolicy definition:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: hazelcast-enterprise-psp
+spec:
+  allowPrivilegeEscalation: false
+  forbiddenSysctls:
+  - '*'
+  fsGroup:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  requiredDropCapabilities:
+  - ALL
+  runAsUser:
+    rule: MustRunAsNonRoot
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    ranges:
+    - max: 65535
+      min: 1
+    rule: MustRunAs
+  volumes:
+  - configMap
+  - emptyDir
+  - secret
+  - persistentVolumeClaim
+```
+
+* Custom ClusterRole for the custom PodSecurityPolicy:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: hazelcast-enterprise-clusterrole
+rules:
+- apiGroups:
+  - extensions
+  resourceNames:
+  - hazelcast-enterprise-psp
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+  ```
+
+### Hazelcast Enterprise License Key
+
+You need to create a secret with the Hazelcast Enterprise License Key:
+
+```bash
+$ kubectl create secret generic hz-enterprise-license-key --from-literal=key=LICENSE-KEY
```

Comment:
Please remove the extra `=` between `literal` and `key`
--from-literal=key=LICENSE-KEY ===> --from-literal key=LICENSE-KEY

Reply:
`kubectl create secret generic hz-enterprise-license-key --from-literal=key=LICENSE-KEY` is actually a correct syntax. You can find similar usages, for example, here: https://github.com/IBM/charts/blob/master/community/artifactory-ha/README.md

Reply:
yes it is indeed. Please ignore my comment. Looks good. Thanks.